R/calcUNFCCC.R

```diff
@@ -239,11 +236,14 @@ calcUNFCCC <- function() {
 
   # return results ----
 
-  # set 0 to NA in calculated variables
-  calculatedVariabes <- setdiff(getNames(x), mappedVariables)
-  tmp <- x[, , calculatedVariabes]
-  tmp[tmp == 0] <- NA
-  x[, , calculatedVariabes] <- tmp
+  # fill countries of selected regions with 0 to allow for regional aggregation
+  regions.fill <- c("EUR", "REF", "NEU", "CAZ")
```

Comment:
can you please also add "EU27" to this list?

Reply:
it should be implicitly included by EUR, as EU27 is a subset of EUR